### PR TITLE
Add Headers parsing function

### LINKED ISSUE #34: langneg: add Accept-Language header parsing support
Since fluent-langneg runs on both server and client, it would be nice to not have to separately parse the Accept-Language header string into an array of locales:

```js
negotiateLanguages(req.header('Accept-Language'), availableLocales)
```

As things stand, I've either got to add another language negotiation library, like accept-language-parser, or roll my own, probably buggy, regex to do the job ;-)

### PR DESCRIPTION
**Pull request by zbraniecki**
Fixes #34

### CODE REVIEW

File: fluent-langneg/src/headers.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+export default function parseAcceptedLanguagesHeaderIntoArray(string = '') {
+  if (typeof string !== 'string') throw new TypeError('Argument must be a string');
+  const tokens = string.split(',').map(t => t.trim());
+  return tokens.filter(t => t !== '').map(t => {
+    return t.split(';')[0];
```
Comment (stasm) [inline fluent-langneg/src/headers.js]: This doesn't take into effect the q-value nor the specificity of the locale tag. Are we fine with this?

File: fluent-langneg/src/headers.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+export default function parseAcceptedLanguagesHeaderIntoArray(string = '') {
```
Comment (stasm) [inline fluent-langneg/src/headers.js]: Rename this simply to `acceptedLanguages` please.

File: fluent-langneg/src/headers.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+  if (typeof string !== 'string') throw new TypeError('Argument must be a string');
```
Comment (stasm) [inline fluent-langneg/src/headers.js]: Please break this into an `if .. ` and a `{}` block.

**Comment by stasm** [comment]
And maybe rename the file to `accepted_languages.js` :)

File: fluent-langneg/src/headers.js (hunk 1 above)
Comment (zbraniecki) [inline fluent-langneg/src/headers.js]: yes, this is on purpose. We don't act upon the q-value at all. The order is the only indication for our fallback, there's no increase/decrease in fallback based on the q-value.

**Comment by zbraniecki** [comment]
Applied feedback

File: fluent-langneg/src/headers.js (hunk 1 above)
Comment (stasm) [inline fluent-langneg/src/headers.js]: Do you plan to extend this in the future? The RFC says the q-values are in fact important.

File: fluent-langneg/src/accepted_languages.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+
+export default function aceptedLanguages(string = '') {
```
Comment (stasm) [inline fluent-langneg/src/accepted_languages.js]: Typo: `s/acepted/accepted/`.

File: fluent-langneg/src/accepted_languages.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+  if (typeof string !== 'string') {
+    throw new TypeError('Argument must be a string');
+  }
+  const tokens = string.split(',').map(t => t.trim());
+  return tokens.filter(t => t !== '').map(t => {
+    return t.split(';')[0];
+  });
```
Comment (stasm) [inline fluent-langneg/src/accepted_languages.js]: This is a nit, but there's no need to use `{}` here and an explicit `return`. Can you change this to a one-liner arrow function?

**Pull request merged**